mediation_service/tests/performance/locustfile.py

```diff
@@ -1,40 +1,24 @@
 import os
-from common.auth import Auth
 from locust import HttpUser, task, between, run_single_user
+from locust.exception import RescheduleTask
+
+RETRY_ON_FAIL = False
 
 
 class LoadTestUser(HttpUser):
     cookie = ""
     wait_time = between(1, 5)
     host = os.environ["SERVICE_BASE_PATH"]
 
-    def auth(self):
```

Comment:
Since the mediation service itself is Application-Authenticated we don't need to call this auth code for each simulated user anymore. I've kept the `auth.py` code itself though since it could come in handy if we want to load test the demo client, or anything that requires an NHS Login during the request process really.